docs/add_dataset.md

```diff
@@ -527,13 +527,46 @@ use the
 [BibTeX Online Editor](https://truben.no/latex/bibtex/) to create a custom
 BibTeX entry (the drop-down menu has an `Online` entry type).
 
-### 4. Add a test
+### 4. Run `generate_fake_examples`
+The dataset test needs fake examples of dataset. This script generate a fake examples.
+
+```bash
+python -m tensorflow_datasets.scripts.generate_fake.examples \
+                                 --dataset_name dataset_name \
+                                 --file_path None \
+                                 --dataset_type dataset_type # default is image
```

Comment:
Yes, it can auto-detect.
file_path is an input path that wants to generated.
With my last commits, if you don't give a dataset_name, it'll generated under the `fake` file that's in the input path directory.